tests/unit/type/stream.tcl

```diff
@@ -168,6 +168,26 @@ start_server {
         assert {[r xrange mystream - +] == [lreverse [r xrevrange mystream + -]]}
     }
 
+    test {XRANGE exclusive ranges} {
+        set ids {0-1 0-18446744073709551615 1-0 42-0 42-42
```

Comment:
we should add more cases where we use the extreme ids with `(`, like `(0-0` and '(18446744073709551615-18446744073709551615`

Reply:
Agreed, added.